runtime/compiler/make.vim

```diff
@@ -0,0 +1,11 @@
+if exists("current_compiler") | finish | endif
```

Comment:
shouldn't it also reset the global variable `g:current_compiler` instead of finishing?